### ISSUE TITLE
code of FR Modules

### ISSUE THREAD

**Issue by HanAccount** [shiming-chen/FREE#2]
Hi! I found  the function forward code of FR class, 'att' is missing.but your
code h=torch.cat((feat, att), dim=1) exists 'att'.

**Comment by shiming-chen** [comment]
Hi， @HanAccount ,
Thank you for interesting in our work. The code `h=torch.cat((feat, att), dim=1)` is used for debugging, it has not been used in our final version. As such, you can directly comment it.

best!

**Comment by HanAccount** [comment]
ok!Will your code be updated to the final version?I intend to implement this code

**Comment by HanAccount** [comment]
thanks! @shiming-chen
